WeScan/Extensions/UIImage+Utils.swift

```diff
@@ -35,4 +40,50 @@ extension UIImage {
         return UIImage(cgImage: croppedImage)
     }
     
+    /// Scales the image to the specified size in the RGB color space.
+    ///
+    /// - Parameters:
+    ///   - scaleFactor: Factor by which the image should be scaled.
+    /// - Returns: The scaled image.
+    func scaledImage(scaleFactor: CGFloat) -> UIImage? {
+        guard let cgImage = self.cgImage else { return nil }
+        
+        let customColorSpace = CGColorSpaceCreateDeviceRGB()
+        
+        let width = CGFloat(cgImage.width) * scaleFactor
+        let height = CGFloat(cgImage.height) * scaleFactor
+        let bitsPerComponent = cgImage.bitsPerComponent
+        let bytesPerRow = cgImage.bytesPerRow
+        let bitmapInfo = cgImage.bitmapInfo.rawValue
+        
+        guard let context = CGContext(data: nil, width: Int(width), height: Int(height), bitsPerComponent: bitsPerComponent, bytesPerRow: bytesPerRow, space: customColorSpace, bitmapInfo: bitmapInfo) else { return nil }
+        
+        context.interpolationQuality = .high
+        context.draw(cgImage, in: CGRect(origin: .zero, size: CGSize(width: width, height: height)))
+        
+        return context.makeImage().flatMap { UIImage(cgImage: $0) }
+    }
+    
+    /// Returns the data for the image in the PDF format
+    func pdfData() -> Data? {
+        // Typical Letter PDF page size and margins
+        let pageBounds = CGRect(x: 0, y: 0, width: 612, height: 792)
```

Comment:
Correct me if I'm wrong but this is the American paper size. The rest of the world uses ISO standard A4 (842 x 595 points). 
See http://www.derbycitylitho.com/paper-sizes.
But more importantly, should we infer the size of the PDF from the size of the rectangle we detected?

Reply:
My bad - I'll update to the A4 size.

That was the original method (before my commits this morning) but that created PDF's of all sizes, which felt weird - this way, all the PDFs will be equally sized and congruent.

Reply:
> Hey @jcampbell05, managed to improve the performance significantly by scaling the image and putting it into an RGB colorspace!
> 
> This PR is now ready for review 🎉

Nice :) When we add it to our app I'll see if we can optimize any further.

Reply:
I think that this is good enough for now, but maybe in the future we could give WeScan's implementers a way to decide which size they want to you.

Reply:
👍